store/sqlstore/channel_store.go

```diff
@@ -796,10 +796,10 @@ func (s SqlChannelStore) SetDeleteAt(channelId string, deleteAt, updateAt int64)
 		// Additionally propagate the write to the PublicChannels table.
 		if _, err := transaction.Exec(`
 			UPDATE
-			    PublicChannels 
-			SET 
+			    PublicChannels
```

Comment:
Can you please remove this unrelated changes? It would be appreciated, if you open a separate PR for this.